be/src/storage/lake/rowset.cpp

```diff
@@ -39,6 +38,9 @@ StatusOr<ChunkIteratorPtr> Rowset::read(const vectorized::Schema& schema, const
     seg_options.chunk_size = options.chunk_size;
     seg_options.global_dictmaps = options.global_dictmaps;
     seg_options.unused_output_column_ids = options.unused_output_column_ids;
+    if (options.delete_predicates != nullptr) {
```

Comment:
delete predicate is not stored in meta? Why to user _index?

Reply:
The index is used to get the delete predicate which version > this rowset.